articles/storage/blobs/blob-storage-monitoring-scenarios.md

```diff
@@ -127,6 +127,8 @@ For the "how" portion of your audit, the `OperationName` field shows which opera
 
 For the "who" portion of your audit, `AuthenticationType` shows which type of authentication was used to make a request. This field can show any of the types of authentication that Azure Storage supports including the use of an account key, a SAS token, or Azure Active Directory (Azure AD) authentication.
 
+A very common scenario is identifying any un-intentional deletes happening on the account. From the above section of splitting the API by name, it shall help identify the "when" as to when the API’s such as Delete Blob, Delete Container etc were called. For the “who” part, if the authentication type is Azure AD, you can check via field `RequesterObjectId` however if the authentication type is account key or SAS, you can then check based on fields such as `callerIpAddress` and `userAgentHeader`.
```

Comment:
```suggestion
If the request is authorized by using Azure AD, you can use the `RequestObjectId` field to identify the "who". Shared Key and SAS authentication provide no means of auditing individual identities. In those cases, the `callerIPAddress` and `userAgentHeader` fields might help you to identify the source of the operation. If a SAS token was used to authorize an operation, you can identify that token, and if you've mapped tokens to token recipients, you can identify which user, organization, or application has performed the operation. See [Identifying the SAS token used to authorize a request](#identifying-the-sas-token-used-to-authorize-a-request).
```

Reply:
Hi @normesta - Thanks for reviewing and proposing the edits to best fit in here.

I have just one suggestion:
For the pointer "and if you've mapped tokens to token recipients", if we can replace it by "and if you've mapped tokens to token recipients at your end" I felt it will be better,

To clarify that mapping is something they need to do at their end. At times there are ask of identifying who used SAS and all so was ensuring to clarify that.

Please review and will be good to close out ahead.